### DISCUSSION TITLE
Support for T-RBG displays from Lilygo (ST7701 driver)

### DISCUSSION THREAD

**Discussion by themagicm** [Bodmer/TFT_eSPI#2591] [New processor/display support or capability request] [answered]
Is it possible to implement this round display with the esp32 chip?

https://www.lilygo.cc/products/t-rgb

**Comment by Bodmer** [comment, marked as answer]
The ST7701 driver is not currently supported.

It is possible to add new drivers but it will be some time before that happens.

**Comment by s54b32julian** [comment]
I would also be very interested in this, if it will happen!

**Reply by s54b32julian** [reply]
Just in case you want to implement this display/driver in your fantastic library, here is the example code from LilyGo https://github.com/Xinyuan-LilyGO/T-RGB/blob/main/examples/GFX/GFX.ino 

and for driver: https://github.com/Xinyuan-LilyGO/T-RGB/blob/main/lib/Arduino_GFX/src/display/Arduino_ST7701_RGBPanel.cpp

I would be a pleasure if this Display will work someday with this lib!
